fileutil/tarball_compressor.go

```diff
@@ -68,10 +70,10 @@ func (c tarballCompressor) CompressSpecificFilesInDir(dir string, files []string
 				return bosherr.WrapError(err, "Resovling relative tar path")
 			}
 
-			if fi.IsDir() {
-				relPath = relPath + "/"
+			header.Name = relPath
+			if runtime.GOOS == "windows" {
+				header.Name = strings.ReplaceAll(relPath, "\\", "/")
```

Comment:
I'm trying to understand how this fixes anything, given that this looks really similar to what https://pkg.go.dev/path/filepath#FromSlash is supposed to be doing:
```
FromSlash returns the result of replacing each slash ('/') character in path with a separator character. Multiple slashes are replaced by multiple separators.
```